clinica/converters/aibl_to_bids/utils/json.py

```diff
@@ -0,0 +1,250 @@
+            ["ReconMethodName", ("ReconstructionMethod",), "n/a"],
+            [
+                "ReconMethodParameterLabels",
+                ("ReconstructionMethod",),
+                [""],
+            ],  # todo : corresp more precise ?
+            ["ReconMethodParameterUnits", ("ReconstructionMethod",), [""]],
+            ["ReconMethodParameterValues", ("ReconstructionMethod",), [""]],
+            ["ReconFilterType", ("ConvolutionKernel",), "n/a"],
+            ["ReconFilterSize", ("ConvolutionKernel",), np.nan],  # todo : corresp ?
+            ["AttenuationCorrection", ("AttenuationCorrectionMethod",), "n/a"],
+        ],
+    ).set_index(keys="BIDSname", drop=False)
+
+
+def _get_dcm_value_from_header(
+    keys_list: Tuple[str], dicom_header: Union[FileDataset, DataElement]
+) -> Union[str, int, list, None]:
+    """
+    Get the value from the dicom header corresponding to a dicom Tag/key list
+
+    Parameters
+    ----------
+    keys_list :
+        Tuple representing where to get the expected information in the dicom header
+
+    dicom_header :
+        Dicom header to parse
+
+    Returns
+    -------
+        The value of the tag obtained from the header
+
+    """
+    # todo : how do i test this with sequences ?
+    if not keys_list:
+        # Handles case where is ()
+        return None
+
+    result = None
+    if len(keys_list) == 1:
+        result = dicom_header.get(Tag(keys_list[0]))
+    elif header := dicom_header.get(Tag(keys_list[0])):
+        # Handles nested tags
+        _get_dcm_value_from_header(keys_list[1:], header[0])
+    if result:
+        return result.value
+
+    # todo
+    # if result:
+    #     if type(result.value) == MultiValue:
+    #         # Handles case where result is MultiValue, which is not JSON serializable
+    #         return list(result.value)
+    #     return result.value
+
+    return None
+
+
+def _update_from_image_dicoms(metadata: pd.DataFrame, dcm_dir: Path) -> None:
+    from pydicom import dcmread
+
+    try:
+        dicom_header = dcmread(next(dcm_dir.rglob("*.dcm")))
+    except IndexError:
+        cprint(
+            msg=f"No DICOM found at {dcm_dir}, the image json will be filled with default values",
+            lvl="warning",
+        )
+    else:
+        # todo : prints in debug ?
+        for index, data in metadata.iterrows():
+            if dcm_value := _get_dcm_value_from_header(data.DCMtag, dicom_header):
+                metadata.loc[index, "Value"] = dcm_value
+
+
+def _set_scan_start(dcm_result: pd.DataFrame) -> None:
+    dcm_result.loc["ScanStart", "Value"] = "0"
+
+
+def _set_injection_start(dcm_result: pd.DataFrame) -> None:
+    if (injection := dcm_result.loc["InjectionStart", "Value"]) and (
+        acquisition := dcm_result.loc["TimeZero", "Value"]
+    ):
+        dcm_result.loc["InjectionStart", "Value"] = injection - float(acquisition)
+
+
+def _get_admin_mode_from_start_time(dcm_result: pd.DataFrame) -> None:
```

Comment:
If you have, for all these small functions, some context or knowledge about why you update things this way, it could be nice to write it in a small docstring.